package.json

```diff
@@ -248,6 +251,7 @@
     "babel-plugin-dynamic-import-node": "1.0.0",
     "babel-plugin-react-intl": "2.3.1",
     "babel-plugin-react-transform": "2.0.2",
+    "babel-plugin-styled-components": "^1.1.4",
```

Comment:
Previous convention was not to use semver symbols like caret (^) in `package.json`